docs/advanced-topics/stack-monitoring.asciidoc

```diff
@@ -85,28 +87,20 @@ spec:
     metrics:
       elasticsearchRefs:
       - name: monitoring
-        namespace: observability <1>
+        namespace: observability <2>
     logs:
       elasticsearchRefs:
       - name: monitoring
-        namespace: observability <1>
+        namespace: observability <2>
 ----
 
-<1> The use of `namespace` is optional if the monitoring Elasticsearch cluster and the monitored Elastic Stack resource are running in the same namespace.
-
-NOTE: You can configure an Elasticsearch cluster to monitor itself.
+<1> The same monitoring cluster is used for metrics and logs, but seperate clusters could be used.
 
-NOTE: If Stack Monitoring is configured for a Beat, but the corresponding Elasticsearch cluster is not monitored, the Kibana Stack Monitoring page will not show the Beats data.
+<2> The use of `namespace` is optional if the monitoring Elasticsearch cluster and the monitored Elastic Stack resource are running in the same namespace.
 
 NOTE: If Logs Stack Monitoring is configured for a Beat, and custom container arguments (`podTemplate.spec.containers[].args`) include `-e`, which enables logging to stderr and disables log file output, this argument will be removed from the Pod to allow the Filebeat sidecar to consume the Beat's log files.
 
-NOTE: Stack Monitoring for Logstash on ECK is only available for Logstash versions 8.7.0 and later.
```

Comment:
I don't see this information anymore, do you want to delete it?

Reply:
In my opinion, it can be removed here. Logstash on ECK is not supported below 8.7.0. [The Logstash page is very clear about that](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-logstash.html). I don't think this note has to be here as you will set up Logstash on ECK just to test the stack monitoring. In other words, Logstash on ECK is done before enabling Stack Monitoring.

Reply:
Makes sense.